test/hca-rjsf/test-data.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Should we turn this into data for schema tests, like the other form? Don't have to do that in this PR, though.